backend/config.json.example

```diff
@@ -1,14 +1,18 @@
 {
     "target_size": [1220,610],
     "tolerance": 0.01,
-    "company_name": "com.nortd.labs",
+    "company_name": "/dev/ttyACM0",
```

Comment:
Hi,

ich finde dass mindestens in der config.json.example alle 
konfigurierbaren Parameter mit ihren Standardwerten drinnstehen sollten 
- das erspart einem das Durchsuchen des Quellcodes falls eine 
Einstellung geändert werden soll.

Grüße,
Philip

On 06.05.2016 16:26, morbidick wrote:
>
> In backend/config.json.example 
> <https://github.com/fablab-ka/LasaurApp/pull/8#discussion_r62336494>:
>
> > @@ -1,14 +1,18 @@
> >  {
> >      "target_size": [1220,610],
> >      "tolerance": 0.01,
> > -    "company_name": "com.nortd.labs",
> > +    "company_name": "/dev/ttyACM0",
>
> Hallo @wkraft-fablabka <https://github.com/wkraft-fablabka>,
> ich hab das mit dem letzten PR ja gemacht, daraus ist soweit auch die 
> config.json.example entstanden. Ich hätte eigtl nur die vernünftigen 
> defaults da drinnen, wie zB das das Ding non-public und ohne RFID 
> läuft. Damit nicht jemand der das Ding cloned erstmal ne Stunde 
> debuggen muss.
> Unsere eigene Config wird dann über das Ansible-Script geliefert [1].
> Gruß
> Lukas
>
> [1] 
> https://github.com/fablab-ka/ansible-flka/blob/master/lasersaur.yml#L15-20
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly or view it on GitHub 
> <https://github.com/fablab-ka/LasaurApp/pull/8/files/1a939fea81d939ce9056d923f01c1b1ef82b5749#r62336494>
>